lms/static/sass/_experiments.scss

```diff
@@ -0,0 +1,248 @@
+// edX LMS - experiments
+// ====================
+// NOTE: use this area for css for ab tests and other experiments.
+// Please list the ticket number of the experiment
+
+// --------------------
+// LEARNER-1312 Track Selection V2
+/* This css was added as part of the LEARNER-1312 experiment */
+.v2.register-choice {
+    margin: 0 2% 20px 0 !important
+}
+.v2.register-choice-certificate .list-actions {
+    text-align: left !important;
+}
+.v2.register-choice-donate .list-actions {
+    margin-bottom: 0 !important;
+}
+.v2.register-choice-donate .action-select {
+    display: inline-block !important;
+    list-style-type: none !important;
+    width: 100% !important;
+}
+.v2.register-choice-donate .donation-link {
+    display: inline-block !important;
+    padding: 10px 15px !important;
+    border-radius: 3px !important;
+    border: 1px solid #D7548E !important;
+    box-shadow: 0 2px 1px 0 #982c62 !important;
+    background: white !important;
+    text-align: center !important;
+    color: #D7548E !important;
+    float: left !important;
+}
+.v2.register-choice-v2-audit {
+    height: 250px !important;
+    background: none !important;
+    border-top-color: grey !important;
+    border-top-width: 1px !important;
+}
+.v2.register-choice-v2-audit .list-actions {
+    margin-bottom: 0 !important;
+}
+.v2.register-choice-v2-audit .list-actions input {
+    background: transparent !important;
+    color: #0075b4 !important;
+    box-shadow: none !important;
+    text-decoration: underline !important;
+    border: none !important;
+}
+.v2.register-choice-v2-audit .wrapper-copy-inline {
+    height: 70px !important;
+    width: 100% !important;
+    display: flex !important;
+}
+.v2.register-choice-v2-audit .wrapper-copy {
+    width: 70% !important;
+    height: auto !important;
+}
+
+.v2.page-header {
+    padding-bottom: 0;
+}
+.v2 img {
+    margin-top: 20px;
+    margin-left: 5px;
+}
+.v2 .donation-link {
+    font-weight: bold !important;
+}
```

Comment:
@MatthewPiatetsky note that because this is Sass you should only need to add ```.v2``` once and can nest the rest of the code within it.